source/blog/2019-05-11-rspec-rails-4-0-has-been-released.md

```diff
@@ -0,0 +1,175 @@
+---
+title: RSpec Rails 4.0 has been released!
+author: Benoit Tigeot, Jon Rowe and Phil Pirozhkov
+---
+
+RSpec Rails 4.0 has been released recently! Given our commitment to
+[semantic versioning](http://semver.org/), this update come with 
+breaking changes for people that are using Rails below 4.2. If we 
+did introduce any bugs, please let us know, and we'll get 
+a patch release out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes 258 commits and 56
+merged pull requests from 27 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+RSpec Rails was release with full support and many improvements for 
+Rails 6.
+
+### Breaking changes
+
+This version no longer support Ruby bellow 2.3. We soft support
+Rails 4.2 but we encourage you to use rspec-rails with Rails 5
+at least.
+
+### JRuby support
+
+We very happy to support JRuby on this new release. You can now use
+`rspec-rails` with Rails 5.2 and 6 and JRuby.
+
+### Support for Action Cable
+
+Using a [channel spec][1] it will be easy to test your Action Cable
+channels. A `channels spec` is a thin wrapper for an
+`ActionCable::Channel::TestCase`, and includes all of the behavior
+and assertions that it provides, in addition to RSpec's own 
+behavior and expectations.
+
+It also includes helpers from `ActionCable::Connection::TestCase`
+to make it possible to test connection behavior.
+
+### Support for Action Mailbox
+
+Rails 6 came with [Action Mailbox][2]. We can now test them using
+`:mailbox` flag.
+
+```ruby
+```
+
+TODO: We do no have feature test for this...
+
+### Improvements on generators
+
+If you use generators, there is some improvements:
+- No more Ruby 1.9 hash syntax in file generated
+- Scaffold generator generates request specs instead of 
+controller spec
+- New [generators][3] available (channel, generator, mailbox...)
+
+### Choose which Active Job `queue_adapteur` in system specs
+
+Some people want to run Active Job with a specific queue. In system
+specs you can now choose your desired queue:
+
+```ruby
+before do
+  ActiveJob::Base.queue_adapter = :inline
+end
+```
+
+### Silent log with Puma
```

Comment:
Feel free to ask or remove this kind of proposal. I do not want too add too many but I think is is important to have a detailed changelog.

Reply:
Those Puma startup logs were very annoying, I would update just because of them being muted :D